profile/index.php

```diff
@@ -210,7 +210,7 @@ function checkCreds(){
 			<div>
 				<div><a href="#" onclick="toggle('emaildiv');"><?php echo htmlspecialchars((isset($LANG['RETRIEVE'])?$LANG['RETRIEVE']:'Retrieve Login'), HTML_SPECIAL_CHARS_FLAGS); ?></a></div>
```

Comment:
Don't know if this is in 508 compliance or not but I couldn't find a nice way to keyboard navigate to this link and the others except for command f the link or ' to quick look up links. Adding tabindex also didn't work because the href is being used as a button so maybe a some kind of button would be best here instead of a link.

Reply:
Interesting. For me, it was this line that I couldn't navigate to via keyboard:
`<div style="color:blue;cursor:pointer;" onclick="resetPassword();"><?php echo (isset($LANG['REST_PWD'])?$LANG['REST_PWD']:'Reset Password'); ?></div>`

Which makes sense; it's not an anchor element with an href attribute. Once I added those in, I was able to keyboard navigate to all of the links. Good catch.

Resolved in commit 80de7c661.